dbus/dbus_libgio.go

```diff
@@ -36,6 +48,34 @@ func (d *dbusAPILibGio) IsGUID(str string) bool {
 	return goBool(C.g_dbus_is_guid(cstr))
 }
 
+// ServerNew creates a new D-Bus server that listens on the specified address
+// https://developer.gnome.org/gio/stable/GDBusServer.html#g-dbus-server-new-sync
+func (d *dbusAPILibGio) ServerNew(address string, flags int) (unsafe.Pointer, error) {
```

Comment:
As discussed in chat, if this is not going to be used in the client itself, but is useful in tests, it would be good to move it into test files or its own package, to avoid importing it in the client and wasting space.

Reply:
moved to dbus/server